static/app/views/detectors/components/forms/metricFormData.tsx

```diff
@@ -0,0 +1,311 @@
+import {useFormField} from 'sentry/components/workflowEngine/form/useFormField';
+import {PriorityLevel} from 'sentry/types/group';
+import type {
+  DataCondition,
+  DataConditionGroup,
+} from 'sentry/types/workflowEngine/dataConditions';
+import {
+  DataConditionGroupLogicType,
+  DataConditionType,
+} from 'sentry/types/workflowEngine/dataConditions';
+import type {Detector} from 'sentry/types/workflowEngine/detectors';
+import {
+  AlertRuleSensitivity,
+  AlertRuleThresholdType,
+} from 'sentry/views/alerts/rules/metric/types';
+
+interface PrioritizeLevelFormData {
+  /**
+   * Issue is created at this priority level
+   */
+  initialPriorityLevel: PriorityLevel;
```

Comment:
will this change depending on the context of the form? could we just say this is the low priority?